tox.ini

```
@@ -46,11 +45,12 @@ deps =
     {[testenv:integrationtests]deps}
     {[testenv:unittests]deps}
     -r dev_requirements/requirements-linting.txt
-    # add your fixtures like e.g. pytest_datafiles here
 setenv = PYTHONPATH = {toxinidir}/src
 commands =
-    #pylint tests --rcfile=tests/.pylintrc
-    # add single files (ending with .py) or packages here
+    tox -e init_bo4e
```

```suggestion
    tox -e init_bo4e
    pylint src/borm
```
da sollte was zusammenkommen ;)

---

6995803f6ea3c7b86de94303f3ddbad5358c4e8b